### DISCUSSION TITLE
v1.25.0

### DISCUSSION THREAD

**Discussion by Quitch** [Legion-Expansion/Legion-Expansion#609] [Announcements]
### General

- Added support for the Bugs' Faction metal extractors

### Balance

- Land
  - Odin
    - Removed stomp weapon
  - Omni Silo
    - Energy storage increased from 75,000 to 225,000
  - Thor
    - Removed stomp weapon
- Orbital
  - Rig
    - Cost increased from 3000 to 4000

### Fixed

- Manta not showing DPS for any weapon on its unit card
- Manta's anti-drop-pod weapon being linked to its missile targeting system
- Manta AA effects not being tied to where it was shooting
- Interplanetary speeds and unittype tags for Infiltrator - with thanks to Dreadnought for the report
- Havoc missing UNITTYPE_TacticalDefense

<hr /><em>This discussion was created from the release <a href='https://github.com/Legion-Expansion/Legion-Expansion/releases/tag/v1.25.0'>v1.25.0</a>.</em>

**Comment by jjjr2062** [comment]


**Reply by Quitch** [reply]
In their respective repos within the Legion Expansion organisation.

**Reply by Quitch** [reply]
I've also added them to the release

**Comment by jjjr2062** [comment]


**Reply by Quitch** [reply]
It's attached to the latest release on this repo.

**Comment by jjjr2062** [comment]


**Reply by Quitch** [reply]
That's why you should get it through Community Mods
